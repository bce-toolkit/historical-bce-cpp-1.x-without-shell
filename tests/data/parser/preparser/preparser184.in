NCl3=N2+Cl2