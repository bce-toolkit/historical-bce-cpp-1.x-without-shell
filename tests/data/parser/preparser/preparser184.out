NCl3 0 0
N2 1 0
Cl2 1 0
